src/mailchimp/mailchimp_manager.py

```diff
@@ -8,13 +8,18 @@
     * Implement methods
     * Implement datalogging
 """
+import mailchimp_marketing as MailchimpMarketing
+
+from mailchimp_marketing.api_client import ApiClientError
 
 from typing import *
 
 from requests import *
 
 from src.attendees.attendee_manager import Attendee, AttendeeManager
 
+import json
+
```

Comment:
Importing libraries should be done as built-in libraries first, empty line, third party libraries, empty line, then project libraries. Please see https://www.python.org/dev/peps/pep-0008/#imports